src/Tests/BaseClassesTests/AttributeTest.java

```diff
@@ -18,7 +18,7 @@ void getValue() {
     void setValue() {
         Attribute<String> attribute = new Attribute<>("yes");
         attribute.setValue("no");
-        assertEquals(attribute.getValue(), "no");
+        assertEquals(attribute.getValue(), "yes"); //TODO: UNBREAK THIS
```

Comment:
This is intentional just to make sure the test results are actually reported well. It will be fixed once everything is confirmed working.